.github/workflows/deploy-go-wr-pr.yml

```diff
@@ -10,19 +10,27 @@ permissions:
 on:
   pull_request:
     paths-ignore:
-      - 'apps/gi-frontend/**'
```

Comment:
I seem to have forgotten to put these in here. Do you think we should add to the commit msg template that if you add a library you should amend these files?

Reply:
I'll do another PR